src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs

```diff
@@ -216,8 +216,6 @@ private void RestorePackages()
                 TraceWriter.Error(ex.Message, ex is CompilationErrorException ? null : ex);
```

Comment:
Need to remove this, since the rethrown error is now automatically logged by the base class